org.opent2t.onboarding.insteon/js/tests/test.js

```diff
@@ -0,0 +1,31 @@
+var test = require('ava');
+var OpenT2T = require('opent2t').OpenT2T;
+var testConfig = require('./testConfig');
+var onboardingPath = require('path').join(__dirname, '..');
+
+console.log("Test Config:");
+console.log(JSON.stringify(testConfig, null, 2));
+
+///
+/// Run a series of tests to validate the translator
```

Comment:
Fixed.

---
In reply to: [82907798](https://github.com/openT2T/onboarding/pull/21#discussion_r82907798) [](ancestors = 82907798)